packages/manager/apps/dedicated/client/app/billing/main/history/legacy/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,29 @@
+{
+  "billing_main_history_legacy_title": "Mes factures",
```

Comment:
File has been copied, moved to a legacy folder for the US and keys has been prefixed with `legacy`

Reply:
Thanks for the explanation, so no changes here ?

Reply:
@FredericEspiau No content change, only added `legacy` to all keys